strawberry/ext/mypy_plugin.py

```diff
@@ -1,9 +1,11 @@
 import re
-import warnings
 from decimal import Decimal
 from typing import Any, Callable, Dict, List, Optional, Set, Tuple, Union, cast
 from typing_extensions import Final
 
+from packaging.version import Version
+from packaging.version import parse as parse_version
```

Comment:
Mypy switched to a a.b.c format now, so I imported this `parse_version` function from packaging, but I'm not convinced we should add a dependency just for this, maybe we can vendorize the function?

What do you think @BryceBeagle @skilkis?

Reply:
Why not parse the version as a tuple? So `0.991` becomes `(0, 991)` and `1.0.0` becomes `(1, 0, 0)`. Then the comparison below can be done with `version < (0, 93)`

If the versions has any extra non int parts in it (e.g. `0.991.extra`), we could force the first 3 tuple elements to be an int, and fill it with `0`. So that would become `(0, 991, 0, "extra"). And the comparison above would still work correct

Reply:
that's a good point, I didn't want to write any logic for the parsing (I'm lazy), and reading online it seems to be suggested to use a library like that one :D

Reply:
I always prefer to not have to reinvent the wheel too :)

Personally I don't have anything against adding that dependency, specially because it is just an extra ~300kb. Maybe some people can look at it as "wow, that's an extra ~300kb, too much" hahaha

Reply:
Since `packaging` is developed and maintained by PyPA I'm all for it! 🚀 I believe some other projects are still using `pkg_resources` which is discouraged by PyPA themselves, so nice find with `packaging`. I didn't know it existed